stdlib/2/__builtin__.pyi

```diff
@@ -29,6 +29,7 @@ class classmethod: pass  # Special, only valid as a decorator.
 class object:
     __doc__ = ...  # type: Optional[str]
     __class__ = ...  # type: type
+    __slots__ = ...  # type: Optional[Union[str, unicode, Iterable[Union[str, unicode]]]]
```

Comment:
nit: since you're already doing a Union, it would be better to say:
`Union[None, str, unicode, Iterable[str], Iterable[unicode]`

Much shorter :)